frontend/test/metabase/scenarios/question/settings.cy.spec.js

```diff
@@ -130,18 +130,48 @@ describe("scenarios > question > settings", () => {
       cy.findByText("Visible columns").click();
       findColumnAtIndex("Products → Category", 5);
 
+      // https://github.com/metabase/metabase/pull/21338#pullrequestreview-928807257
+
+      // Add "Address"
+      cy.findByText("Address")
+        .siblings(".Icon-add")
+        .click();
       /**
        * Helper functions related to THIS test only
        */
```

Comment:
Nit: This should've been moved down, just above the helper functions, but please don't bother with it now. No point in waiting for the whole CI once again just because of that.